code/script.js

```diff
@@ -1,47 +1,103 @@
 // All the DOM selectors stored as short variables
-const chat = document.getElementById('chat')
+const chat = document.getElementById ('chat')
+const form = document.getElementById ('form')
+const nameInput = document.getElementById ('input-value')
+const inputWrapper = document.getElementById ('input-wrapper')
+
 
 // Global variables, if you need any, declared here
 
-// Functions declared here
+const topicChoice = `
+<form class='topic-form' id='topic-form'>
+  <select id='select'>
+    <option value = '' selected disabled> Click here to chose subject</option>
+    <option id='topicOption' value = 'My childhood'>My Childhood</option>
```

Comment:
If you set the value to `childhood/mother/partner` you could use it in the template for the next bot-message and your code would be more DRY 😸